doc/cephfs/kernel.rst

```diff
@@ -102,6 +102,9 @@ non-default FS on your local FS as follows::
 
     mount -t ceph :/ /mnt/mycephfs2 -o name=fs,mds_namespace=mycephfs2
 
+NOTE: if mounting fails with EHOSTUNREACH that means no MDS is up or the cluster
+is laggy.
+
```

Comment:
This seems somewhat out of place. We don't go into detail about other errors in this document, and the user won't generally see the EHOSTUNREACH anyway. That's just what the mount() syscall returns.

I'd just drop this.

Reply:
Okay, will drop it.